appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js

```diff
@@ -34,6 +34,60 @@ describe('manage docs on appellant case', () => {
 		});
 	});
 
+	it('upload doc and remove when final version', () => {
+		cy.createCase().then((caseRef) => {
+			happyPathHelper.uploadDocAppellantCase(caseRef);
+			caseDetailsPage.clickAddAdditionalDocs();
+			caseDetailsPage.uploadSampleFile(sampleFiles.document);
+			caseDetailsPage.checkFileNameDisplays('sample-file.doc');
+			caseDetailsPage.clickButtonByText('Continue');
+			caseDetailsPage.clickButtonByText('Confirm');
+			caseDetailsPage.clickButtonByText('Confirm');
+			caseDetailsPage.checkAdditonalDocsAppellantCase('sample-file.doc');
+			caseDetailsPage.clickManageAdditionalDocs();
+			caseDetailsPage.clickLinkByText('View and edit');
+			caseDetailsPage.checkCorrectAnswerDisplays('Version', '1');
+			caseDetailsPage.checkCorrectAnswerDisplays('Redaction status', 'No redaction required');
+			caseDetailsPage.clickButtonByText('Remove current version');
+			caseDetailsPage.selectRadioButtonByValue('Yes');
+			caseDetailsPage.clickButtonByText('Continue');
+			caseDetailsPage.checkAdditonalDocsAppellantCase('No documents available');
+		});
+	});
+
+	it('remove doc when not final version', () => {
+		cy.createCase().then((caseRef) => {
+			happyPathHelper.uploadDocAppellantCase(caseRef);
+			caseDetailsPage.clickAddAdditionalDocs();
+			caseDetailsPage.uploadSampleFile(sampleFiles.document);
+			caseDetailsPage.checkFileNameDisplays('sample-file.doc');
+			caseDetailsPage.clickButtonByText('Continue');
+			caseDetailsPage.clickButtonByText('Confirm');
+			caseDetailsPage.clickButtonByText('Confirm');
+			caseDetailsPage.checkAdditonalDocsAppellantCase('sample-file.doc');
```

Comment:
For all of the lines which reference 'sample-file.doc' you could use our new method - (sampleFiles.doc) - add in the variable at the top of the spec file before the first 'it'. variable is let sampleFiles = caseDetailsPage.sampleFiles;

Reply:
I've updated the relevat method to use the variable (sampleFiles.document)